rust/repositories.bzl

```diff
@@ -93,11 +93,14 @@ def BUILD_for_stdlib(target_triple):
     return """
 filegroup(
     name = "rust_lib-{target_triple}",
-    srcs = glob([
-        "lib/rustlib/{target_triple}/lib/*.rlib",
-        "lib/rustlib/{target_triple}/lib/*{dylib_ext}",
-        "lib/rustlib/{target_triple}/lib/*{staticlib_ext}",
-    ]),
+    srcs = glob(
+        [
+            "lib/rustlib/{target_triple}/lib/*.rlib",
+            "lib/rustlib/{target_triple}/lib/*{dylib_ext}",
+            "lib/rustlib/{target_triple}/lib/*{staticlib_ext}",
+        ],
+        allow_empty = True,
```

Comment:
In which case does that glob is empty? Can a comment be added to justify it?

Reply:
That's the error message I got:
`glob pattern 'lib/rustlib/x86_64-unknown-linux-gnu/lib/*.a' didn't match anything, but allow_empty is set to False.`

I don't know if it happens in other cases too.

Reply:
That seems to indicate a potential bug, this is expected that there is no .a file here. I guess the empty glob is acting if any of those is empty.

Can you just add a comment with the pattern that fails to justify this allow_empty? It's not important to fix at the moment but keeping track of why we did that is useful.

Reply:
Added comment

Reply:
Thanks!